compiler/rustc_error_codes/src/error_codes/E0542.md

```diff
@@ -0,0 +1,47 @@
+A feature since is missing.
```

Comment:
```suggestion
The `since` value is missing in a stability attribute.

```

Reply:
Agree, I simply wanted to keep the consistency with this similar one: https://github.com/rust-lang/rust/blob/master/compiler/rustc_error_codes/src/error_codes/E0546.md

Anyway, I can make a new PR later to change it :).